### PR TITLE
Add webpack / TypeScript demo.

### PR DESCRIPTION
**Pull request by prideout**

### CODE REVIEW

File: web/samples/webpack/src/app.ts [hunk 1]
```diff
@@ -0,0 +1,56 @@
+import * as Filament from "filament";
```
Comment (romainguy) [inline web/samples/webpack/src/app.ts]: Copyright header?

File: web/samples/webpack/webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,39 @@
+const path = require('path');
```
Comment (romainguy) [inline web/samples/webpack/webpack.config.js]: Copyright header?

File: web/samples/webpack/webpack.config.js [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+const CopyWebpackPlugin = require('copy-webpack-plugin');
+
+'use strict';
+
+module.exports = {
+    devtool: 'source-map',
+    entry: './src/app.ts',
+    externals: {
+        fs: 'fs',
+        crypto: 'crypto',
```
Comment (romainguy) [inline web/samples/webpack/webpack.config.js]: 2 `crypto`? Also, `crypto`? I assume it has to do with webpack.

Comment (prideout) [reply web/samples/webpack/webpack.config.js]: Yikes good catch, also added explanatory comment

File: web/samples/webpack/webpack.config.js (hunk 2 above)
Comment (prideout) [reply web/samples/webpack/webpack.config.js]: Fixed

File: web/samples/webpack/src/app.ts (hunk 1 above)
Comment (prideout) [reply web/samples/webpack/src/app.ts]: Fixed

**Pull request merged**